pubtools/_quay/iib_operations.py

```diff
@@ -149,26 +166,27 @@ def _sign_index_image(
     to_sign_entries = _index_image_to_sign_entries(
         built_index_image, namespace, dest_tags, signing_keys, target_settings
     )
-    current_signatures = []
+    current_signatures: list[Any] = []
     for signer in target_settings["signing"]:
         if signer["enabled"]:
             signercls = SIGNER_BY_LABEL[signer["label"]]
             signer = signercls(config_file=signer["config_file"], settings=target_settings)
             signer.sign_containers(to_sign_entries, task_id=task_id)
+    # this always returns an empty list
```

Comment:
yes this probably should be fixed as I didn't originally implement removal of index images signatures. I'll fix it in next PR